moped-editor/src/layouts/DashboardLayout/NavBar/DropdownMenu.js

```diff
@@ -56,12 +57,20 @@ export const helpItems = [
   },
 ];
 
-export const arcGISLink = {
-  linkType: "external",
-  link: "https://austin.maps.arcgis.com/apps/webappviewer/index.html?id=404d31d56b57491abe53ccfd718fcaee",
-  title: "Moped map",
-  Icon: <MapOutlinedIcon fontSize="small" />,
-};
+export const analysisItems = [
```

Comment:
nice to see this turn into its own section. Thanks again for remembering about the mobile menu! 🙏 🙌